### PR TITLE
add check pipenv for linux

### PR DESCRIPTION
**Pull request by systemshift**
# Pull Request

## Description
Please include a summary of the change and which issue is fixed. If this closes one or more open issues, [please tag them as described here](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).

## Affected Dependencies
List any dependencies that are required for this change.

## Type of Change
Please mark options that are relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Documentation (non-breaking change which adds documentation)
- [ ] Improvement (non-breaking change that improves the performance or reliability of existing functionality)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## How has this been tested?
- Describe the tests that you ran to verify your changes.
- Provide instructions so we can reproduce.
- List any relevant details for your test configuration.

## Checklist
- [ ] I did follow the [Contribution Guidelines](https://github.com/OpenMined/.github/blob/master/CONTRIBUTING.md)
- [ ] I did follow the [Code of Conduct](https://github.com/OpenMined/.github/blob/master/CODE_OF_CONDUCT.md)
- [ ] I have commented my code following the [OpenMined Styleguide](https://github.com/OpenMined/.github/blob/master/STYLEGUIDE.md)
- [ ] I have added tests for my changes

## Additional Context
Please also include relevant motivation and context.

### CODE REVIEW

**Review by chinmayshah99 — APPROVED** [review]

File: prereqs_linux.sh [hunk 1]
```diff
@@ -20,6 +20,14 @@ else
     sudo apt-get install python3.6
 fi
 
+# pipenv
+echo "Checking for pipenv"
+if python -c "import pipenv" &> /dev/null; then
```
Comment (chinmayshah99) [inline prereqs_linux.sh]: In Linux systems with ubuntu, both python 2 and 3 are installed. 
`python` usually resolute to python2 and python3  is for python 3. 
We need to check which python version are we checking and then accordingly install it using pip

File: prereqs_linux.sh [hunk 2]
```diff
@@ -20,6 +20,14 @@ else
...
+    echo "pipenv is already installed"
+else
+    echo "installing pipenv"
+    pip install pipenv
```
Comment (chinmayshah99) [inline prereqs_linux.sh]: Same applies for choosing between pip and pip3.

**Review by chinmayshah99 — CHANGES_REQUESTED** [review]

File: prereqs_linux.sh [hunk 3]
```diff
@@ -22,11 +22,11 @@ fi
 
 # pipenv
 echo "Checking for pipenv"
-if python -c "import pipenv" &> /dev/null; then
+if python3 -c "import pipenv" &> /dev/null; then
```
Comment (chinmayshah99) [inline prereqs_linux.sh:27]: If you just have python3 installed, does this work?
I suspect the env variable will be just `python` variable, and maybe `python3 `variable will be absent. 
Not sure, needs to be investigated. @replomancer thoughts?

File: prereqs_linux.sh [hunk 4]
```diff
@@ -9,9 +9,9 @@ else
     sudo apt-get install g++
 fi
 
-# python 3.6
+# checking for python 3.6
 echo "Checking for python3 installation"
-if command -v python3 &>/dev/null; then
+if command python --version | grep -q 'Python 3'; then
```
Comment (chinmayshah99) [inline prereqs_linux.sh]: This should be added as another condition check ( else if condition)

**Review by chinmayshah99 — CHANGES_REQUESTED** [review]

**Review by chinmayshah99 — APPROVED** [review]

**Review by codeboy5 — APPROVED** [review]

**Review by chinmayshah99 — APPROVED** [review]

**Pull request merged**